src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/chapterPage/model/ChapterPageModel.kt

```diff
@@ -0,0 +1,41 @@
+package org.wycliffeassociates.otter.jvm.app.ui.chapterPage.model
+
+import io.reactivex.subjects.PublishSubject
+
+class ChapterPageModel {
+    var activeChapter = PublishSubject.create<Int>()
+    var selectedTab = PublishSubject.create<String>()
```

Comment:
Should the `PublishSubject`s be in the model or in the view model? Model keeps the raw state, view model links via observable with the view?